### ISSUE TITLE
Specifying the category you want, auto download all images in one folder

### ISSUE THREAD

**Issue by omarmoussa69** [Mewzax/Hcaptcha-Images-Scraper#2]
Hey, can you make an option that I can specify for example the new "domestic cat" category or "seaplane" or whatever, and an option to download every image so that it will be faster to collect images and sort them manually.

**Comment by Mewzax** [comment]
u want to skip when the question is not the one you want ?

and for the option to download every images you can use old version from:
https://github.com/Mewzax/Hcaptcha-Images-Scraper/commits/ 

just [here](https://github.com/Mewzax/Hcaptcha-Images-Scraper/commit/8741e7577af1b16529bbad6e26fd08ebdbc727a5#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1)

**Comment by omarmoussa69** [comment]
Thanks!

**Comment by omarmoussa69** [comment]
yes exactly I want to skip any question except "seaplanes" for example. 

> u want to skip when the question is not the one you want ?
> 
> and for the option to download every images you can use old version from: https://github.com/Mewzax/Hcaptcha-Images-Scraper/commits/
> 
> just [here](https://github.com/Mewzax/Hcaptcha-Images-Scraper/commit/8741e7577af1b16529bbad6e26fd08ebdbc727a5#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1)

**Comment by Mewzax** [comment]
Its easy, just add if question == "what u want" download else no
